src/DeviceInterface/AndroidInterface.js

```diff
@@ -19,7 +19,9 @@ class AndroidInterface extends InterfacePrototype {
      * @override
      */
     createUIController () {
-        return new NativeUIController()
+        return new NativeUIController({
+            onPointerDown: (event) => this._onPointerDown(event)
```

Comment:
This used to live in the Apple controller only. I've moved it to the prototype for now, but I feel like we're golfing this stuff around. It was very hard to understand where it was living, whose responsibility it was. To be honest I was about to move the handler in the `listenForFormSubmission` file where we have all the other listeners for the form submission, but I ran out of time. We can discuss about it. I understand the need for these listeners to be in the controllers, but also I noticed that for form submission we don't ever access anything else except the `forms` set, so keeping them separate may make sense and we can simplify the controllers. Not entirely sure about it.